code/modules/research/designs.dm

```diff
@@ -45,3 +45,4 @@ other types of metals and chemistry for reagents).
 	var/list/reagents_list = list()			//List of reagents. Format: "id" = amount.
 	var/maxstack = 1
 	var/lathe_time_factor = 1			//How many times faster than normal is this to build on the protolathe
+	var/syndi_allowed = TRUE			//If false, syndi lathes will not be able to make this (used for radios, etc)
```

Comment:
Not sure how I feel about designs being snowflaked in this manner. Would be much happier with removing the specific designs from `known_designs` since its only 3. This can easily be accomplished with

```dm
known_designs["intercom_electronics"] = null // No intercom electronics for syndies
```

Better yet, do this in the `/New()` for the `/datum/research/autholathe/syndicate` datum